students/jemielity_kamil/lesson_07_string_datetime/date_calculator.py

```diff
@@ -0,0 +1,10 @@
+from datetime import datetime, timedelta
+
+year = int(input('Years to add: '))
+day = int(input('Days to add: '))
+hour = int(input('Hours to add: '))
+minute = int(input('Minutes to add: '))
+
+current_date = datetime.now() + timedelta(weeks=year * 52, days=day,
```

Comment:
Just for your information, please avoid the 'current' word in variable name for something custom and to be placed in the future. It makes more sense for the following  statement 'current_date = datetime.now()'